### PR TITLE
Tradingeconomics WS refactor

### PR DESCRIPTION
**Pull request by aalu1418**
Tradingeconomics
- [x] Initial code
- [x] Testing - WS works (but tested outside of market hours)
- [x] Documentation

### CODE REVIEW

File: packages/sources/tradingeconomics/src/config.ts [hunk 1]
```diff
@@ -11,27 +13,65 @@ import { util } from '@chainlink/ea-bootstrap'
...
+      },
+      subscribe: input => {
+        const validator = new Validator(input, customParams)
+        if (validator.error) {
+          return
+        }
+        const base = validator.overrideSymbol(NAME).toUpperCase()
+        return getSubscription(base)
+      },
+      unsubscribe: () => '',
```
Comment (krebernisak) [inline packages/sources/tradingeconomics/src/config.ts:68]: Does the whole connection need to be torn down to unsubscribe? I don't understand these API providers and why they don't handle unsubscribe messages.

**Review by krebernisak — COMMENTED** [review]
Looks good, but please rebase on top of the latest changes from @RodrigoAD, there will be slight changes to the handler.

File: packages/sources/tradingeconomics/src/config.ts (hunk 1 above)
Comment (aalu1418) [reply packages/sources/tradingeconomics/src/config.ts:68]: Yeah...I couldn't find an unsubscribe command in their SDK:
https://github.com/tradingeconomics/tradingeconomics/tree/master/nodejs/stream-nodejs

or in the one that Jonas made either :(

Comment (krebernisak) [reply packages/sources/tradingeconomics/src/config.ts:68]: Ok, then make this `undefined` when Rodrigo makes the underlying changes.

File: packages/sources/tradingeconomics/src/config.ts [hunk 2]
```diff
@@ -11,27 +13,68 @@ import { util } from '@chainlink/ea-bootstrap'
...
+      },
+      unsubscribe: () => undefined,
+      subsFromMessage: (message) => getSubscription(message?.s),
+      isError: (message: any) => Number(message.TYPE) > 400 && Number(message.TYPE) < 900,
+      filter: message => {
+        return message.topic && message.topic !== 'keepalive'
+      },
+      toResponse: (wsResponse: any): AdapterResponse => {
+        wsResponse = { ...wsResponse, result: wsResponse?.price }
+        return Requester.success(undefined, { data: wsResponse })
```
Comment (RodrigoAD) [inline packages/sources/tradingeconomics/src/config.ts:76]: Same as `Intrinio` PR

**Comment by RodrigoAD** [comment]
LGTM. Feel free to merge if acceptance testing is ok

**Pull request merged**